public_html/wp-content/themes/pattern-directory/single-wporg-pattern.php

```diff
@@ -9,7 +9,25 @@
 
 namespace WordPressdotorg\Theme;
 
+use const WordPressdotorg\Pattern_Directory\Pattern_Flag_Post_Type\POST_TYPE;
+
 get_header();
+
+$user_has_reported = false;
+
+if ( is_user_logged_in() ) {
+	$args = array(
+		'author' => get_current_user_id(),
+		'post_parent' => get_the_ID(),
+		'post_type' => POST_TYPE,
+		'post_status' => 'any',
```

Comment:
```suggestion
		'post_status' => 'pending',
```

I think this should probably just check for flags that haven't been reviewed yet.